### DISCUSSION TITLE
blog/one-stop-starter

### DISCUSSION THREAD

**Discussion by giscus** [theodorusclarence/theodorusclarence.com#247] [General]
# blog/one-stop-starter

Increase your efficiency by using preconfigured starter repository, with rich development features and automations.

https://theodorusclarence.com/blog/one-stop-starter

**Comment by theodorusclarence** [comment]
Hello, kindly leave a like (or 5 😄), and comment if you got something to ask! 🚀

**Reply by flosrn** [reply]
Thank you very much for your amazing work, it's the dream Next.js starter!!! Fit perfectly to my needs 🔥

**Comment by supryantowp** [comment]
how to change the font with Google Font, how do you do it?

**Reply by supryantowp** [reply]
ahh  just change in tailwind.config.js

**Comment by tombelieber** [comment]
This is the most awesome thing I've ever seen in React ecosystem! Shout up to you, thank you so much for contributing! Can't wait to try

**Reply by theodorusclarence** [reply]
Thank you for checking it out!

**Comment by PawarAmit52** [comment]
Thanks for bringing this content for us as NextJs and Tailwind CSS are an amazing combination, both [possessing great UI Features](https://www.esparkinfo.com/top-react-js-ui-frameworks.html). NextJS as well as Tailwind CSS is used widely for effective and appealing UI Solutions.

**Comment by gdrkbones** [comment]
Oh right, you also don't have to wait that long because husky only the code that you stage (using 
lint-staged
).

Is this a complete idea? Or there's something missing?

**Comment by gdrkbones** [comment]
I have been playing around with this template and i think its awesome. Could you make an article about types? I mean, components are created in a way i don't really understand. Maybe not an article but i would apreciate any hint on this topic

**Comment by calense** [comment]
This is so cool.
I try to add Material UI in this project, but loos like it cannot find MUI components when I try to type things (such as Box) 
It also cannot auto import Nextjs component when I type it (such as Image)
Could you advise please?

**Comment by returnvoid** [comment]
Thanks for sharing these awesome boilerplate. Question, is there any convention you suggest to add pages using SSR? Also, is the SEO component prepared to use it with dynamic data?
Thanks!

**Comment by danivickym** [comment]
can i use this for new App folder?

**Comment by armageAR** [comment]
Nice starter! Thanks!!!👌👌👌
